doc/classes/GraphNode.xml

```diff
@@ -260,6 +260,11 @@
 				Emitted when the GraphNode is moved.
 			</description>
 		</signal>
+		<signal name="slot_updated">
+			<description>
```

Comment:
You need to add the signal argument here as well. May also be worth linking to the `set_slot` method.